src/main/kotlin/com/hedvig/underwriter/serviceIntegration/productPricing/dtos/CalculateBundleInsuranceCostRequest.kt

```diff
@@ -0,0 +1,5 @@
+package com.hedvig.underwriter.serviceIntegration.productPricing.dtos
+
+data class CalculateBundleInsuranceCostRequest(
+    val productsToBeBundled: List<CalculateBundledPriceDto>
```

Comment:
Either

```suggestion
    val insurancesToBeBundled: List<CalculateBundledPriceDto>
```
or 
```suggestion
    val contractsToBeBundled: List<CalculateBundledPriceDto>
```
Or something, products is kind of a dead concept